easybuild/tools/filetools.py

```diff
@@ -681,6 +681,14 @@ def search_file(paths, query, short=False, ignore_dirs=None, silent=False, filen
     return var_defs, hits
 
 
+def is_empty_folder(folder_path):
```

Comment:
* I would prefer `directory` over `folder` (we don't use `folder` anywhere else)
* A directory containing an empty subdirectory is not really empty, so this should be renamed...
  How about `dir_contains_files` (and flipping the `True`/`False` too, of course)?

Reply:
Renaming makes sense.

I was thinking about future extensions, e.g. adding parameters like `non_empty_if_contains_subdir` or so. But I didn't see it necessary, at least ATM. I still stand by the name: A directory containing only directories is still empty. But I see how your name avoids the ambiguity. Changing...